src/comm/PlanckListener.cc

```diff
@@ -55,3 +64,43 @@ void PlanckListener::onHealthTimeout()
     health_timer.stop();
 }
 
+void PlanckListener::setCharger(bool on)
+{
+    charger_on = on;
```

Comment:
maybe we should send a `chargerRequest()` instead of setting directly the charger state so it doesn't go out of sync with the actual state of the charger

Reply:
Done.